pulsar-flink/pom.xml

```diff
@@ -42,7 +42,7 @@
 
     <dependency>
       <groupId>org.apache.flink</groupId>
-      <artifactId>flink-table_${scala.binary.version}</artifactId>
+      <artifactId>flink-table-planner_${scala.binary.version}</artifactId>
```

Comment:
have you verified that the current flink connectors are working well in 1.8.0? If not, I would suggest leaving flink version unchanged for now, until you verify it.

Reply:
reverted